V5.17 documentation (#3161)
mattermost/docs: source/administration/changelog.md

  - Original 5.17.0 release

### Bug Fixes
 - Fixed an issue where saving an empty string on Marketplace URL reset the URL instead of showing an error.
 - Fixed an issue where the default permission was such that all users were allowed to invite a guest instead of only System Admins.
 - Fixed an issue where Guest users were shown in the list when adding new members to a channel.
 - Fixed an issue where attempting to configure uninstalled plugins got stuck at "Loading..." without timeout.
 - Fixed an issue where clicking "Search" icon in narrow-width mode caused right-hand side to appear along with loading indicator "...".
 - Fixed an issue where ``@all`` notification was still sent to all users when using TAB to press Cancel on the notification prompt.
 - Fixed an issue where system messages could trigger mentions for username collisions.
 - Fixed an issue where code syntax was not rendering or highlighting as expected in markdown.
 - Fixed an issue where users were not able to attach a file from iPad using Safari.
 - Fixed an issue where ``/code`` was rendering HTML incorrectly.
 - Fixed an issue where clicking "Pinned" icon removed text in the search box.

>  - Fixed an issue where **Main Menu > Integrations > OAuth2 Applications** page user interface broke when shrinking the window to a small size.

Reviewer: ```suggestion
 - Fixed an issue where **Main Menu > Integrations > OAuth 2.0 Applications** page user interface broke when shrinking the window to a small size.
```

>  - Fixed an issue where there was no feedback on mobile view that the maximum post length had been exceeded.

Reviewer: ```suggestion
 - Fixed an issue where no feedback was given on mobile view when the maximum post length had been exceeded.
```

>  - Fixed an issue where dragging or dropping a folder should scroll the user to the right-hand side text box to make the error more visible.

Reviewer: ```suggestion
 - Fixed an issue where dragging or dropping a folder did not scroll the user to the right-hand side text box to make the error more visible.
```

 - Fixed an issue on mobile browser view where the post menu was split in 2 and users were not able to scroll up to see "Add Reaction" option.
 - Fixed an issue where pressing and holding on teams and channels in the left-hand side opened the context menu on the Desktop App.
 - Fixed an issue where the user popover bled off screen when browser or Desktop App was set to full-screen mode.
 - Fixed an issue where clicking locally installed plugins without a URL opened a new tab to the same page.
 - Fixed an issue where interactive message buttons and menus were not vertically the same size.
 - Fixed an issue where the first element was selected by default in radio elements in interactive buttons.
 - Fixed an issue where search with quotation marks was not returning expected results.
 - Fixed an issue where bulk importer generated invalid passwords for the user object with a missing password key.
 - Fixed an issue where post metadata was returned for deleted posts.
 - Fixed an issue where users were not able to use ``api/v4/websocket`` with a trailing slash.
 - Fixed an issue with subpaths where in-app System Console links were missing in the ``/subpath`` and resulted in a 404 error.

>  - Fixed an issue where **Terms of Service** and **Privacy Policy** in **Main Menu > About Mattermost** should permanently link to Mattermost's policies.

Reviewer: ```suggestion
 - Fixed an issue where **Terms of Service** and **Privacy Policy** in **Main Menu > About Mattermost** did not permanently link to Mattermost's policies.
```
